dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -10,22 +10,21 @@ class Pd::WorkshopMailer < ActionMailer::Base
   # Name of partial view for workshop details organized by course, then subject.
   # (views/pd/workshop_mailer/workshop_details/_<name>.html.haml)
   DETAILS_PARTIALS = {
-    Pd::Workshop::COURSE_CS_IN_S => {
-      Pd::Workshop::SUBJECT_CS_IN_S_PHASE_2 => 'phase_2',
-      Pd::Workshop::SUBJECT_CS_IN_S_PHASE_3_SEMESTER_1 => 'cs_in_s_phase_3_semester_1',
-      Pd::Workshop::SUBJECT_CS_IN_S_PHASE_3_SEMESTER_2 => 'cs_in_s_phase_3_semester_2'
+    Pd::Workshop::COURSE_CSF => {
+      Pd::Workshop::SUBJECT_CSF_101 => 'csf_intro',
+      Pd::Workshop::SUBJECT_CSF_201 => 'csf_deepdive'
```

Comment:
This is making me wonder if we need something here for the district workshops. I did not get a chance to check the emails for that.

Reply:
Hmm, nothing in https://staging-studio.code.org/rails/mailers/pd_workshop_mailer has the district workshops.

I did a quick Jira search and couldn't find anything about mailers for district workshops (https://codedotorg.atlassian.net/browse/PLAT-1836 added them)

Reply:
AH, you asked this question earlier, and it's the same as other CSF subjects (verified with Tess in a DM)! I'll add this one subject as well to make that clear.

Reply:
(May 26 was when this was confirmed)

Reply:
Let me check in with Danielle on what she wants for emails. The post workshop email is working because I was able to test that. Not sure about the 3 and 10 day. Thanks!